recipes-ni/ni-cgroups/files/ni-cgroups

```diff
@@ -10,6 +10,16 @@
 
 identify_lvrt_cgroup_version()
 {
+	# ARM architectures always use cgroups v1
```

Comment:
Can you reword this to something like `We only support cgroups v1 on ARM for now` for accuracy.